memlab.bib

```diff
@@ -9852,6 +9853,15 @@ @incollection{MannEtal15
 	Title = {The role of context in episodic memory},
 	Year = {2015}}
 
+@incollection{AherBeat81,
+    Title = {{Physiological Evidence that Demand for Processing Capacity Varies with Intelligence}},
+    Year = {1981},
+    Booktitle = {Intelligence and Learning},
+    Author = {Ahern, S and Beatty, J},
+    Pages = {121-128},
```

Comment:
Can you use double dashes for page ranges?  E.g. this should be "121--128"

Reply:
got it!